test/rql_test/test-runner

```diff
@@ -30,7 +30,7 @@ try:
     openFileLimit = int(os.getenv('OPEN_FILE_LIMIT', 512))
     if resource.getrlimit(resource.RLIMIT_NOFILE)[0] < openFileLimit:
         resource.setrlimit(resource.RLIMIT_NOFILE, (openFileLimit, openFileLimit))
-except Exception, e:
+except Exception as e:
```

Comment:
This was in preparation for `tox -e py34` but this file isn't quite py34 compatible but I'll send a separate PR for that.